_posts/2023-07-06-optimize-opensearch-index-shard-size.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title:  "Optimize OpenSearch index shard sizes"
+authors:
+  - ev2900
+  - ronmiller
+date: 2023-07-06
+categories:
+  - technical-posts
+---
```

Comment:
Please add the following meta: 

**Meta_keywords:**  OpenSearch index, OpenSearch shards, shard optimization, optimize shard sizes OpenSearch
**Meta_description:**  Learn how optimizing the number of primary and replica shards in your OpenSearch index can help you improve search latency and write performance.

Reply:
Done. Thanks for keywords and description